.github/workflows/check-pull-request.yml

```diff
@@ -0,0 +1,36 @@
+name: Java CD with Gradle
+
+on:
+  pull_request:
+    branches:
+      - develop
+
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 1
+
+      - name: Set up Java
+        uses: actions/setup-java@v4
+        with:
+          distribution: 'adopt'
+          java-version: '17'
+          cache: 'gradle'
+
+      - name: Build using Gradle Wrapper
+        run: ./gradlew build --no-daemon
+
+  test:
```

Comment:
This job requires a checkout step too.

![image](https://github.com/Pending-Name-21/frontend-library/assets/64939916/fbcfacab-a990-41cd-a8ef-f561be14bf29)

Reply:
Yeah, I forgot to add it. Now it's there.